config.txt

        module.exports = {
            bot: {
                token: "ODc2MDYwOTEzODczMDgwMzkx.YRelTg.D2m5s03faI_YT9tkaniMfYpUfu8",
                prefix: ".",
                owners: ["844244033525514280"],
                mongourl: "mongodb+srv://akg:akg@cluster0.kyx6x.mongodb.net/myFirstDatabase?retryWrites=true&w=majority",
                servers: {
                    token: "ODc2MDYwOTEzODczMDgwMzkx.YRelTg.D2m5s03faI_YT9tkaniMfYpUfu8",
                    prefix: "."
                }
            },
        
            website: {
                callback: "",
                secret: "QwfZu7fr0cdkdmKwJ_-VhtaECQd89gKP",
                clientID: "876060913873080391", // Bot client id.
                tags: [ "Moderation", "Fun", "Minecraft","Economy","Guard","NSFW","Anime","Invite","Music","Logging", "Web Dashboard", "Reddit", "Youtube", "Twitch", "Crypto", "Leveling", "Game", "Roleplay", "Utility", "Turkish" ],
                languages: [
                    { flag: 'gb', code: 'en', name: 'English' },
                    { flag: 'tr', code: 'tr', name: 'Türkçe' },
                    { flag: 'de', code: 'de', name: 'Deutsch' }
                ],
                servers: {
                    tags: [
                    {
                        icon: "fal fa-code",
                        name: "Development"
                    },
                    {
                        icon: "fal fa-play",
                        name: "Stream"
                    },
                    {
                        icon: "fal fa-camera",
                        name: "Media"
                    },
                    {
                        icon: 'fal fa-building',
                        name: 'Company'
                    },
                    {
                        icon: 'fal fa-gamepad',
                        name: 'Game'
                    },
                    {
                        icon: 'fal fa-icons',
                        name: 'Emoji'
                    },
                    {
                        icon: 'fal fa-robot',
                        name: 'Bot List'
                    },
                    {
                        icon: 'fal fa-server',
                        name: 'Server List'
                    },
                    {
                        icon: 'fal fa-moon-stars',
                        name: 'Turkish'
                    },
                    {
                        icon: 'fab fa-discord',
                        name: 'Support'
                    },
                    {
                        icon: 'fal fa-volume',
                        name: 'Sound'
                    },
                    {
                        icon: 'fal fa-comments',
                        name: 'Chatting'
                    },
                    {
                        icon: 'fal fa-lips',
                        name: 'NSFW'
                    },
                    {
                      icon: "fal fa-comment-slash",
                      name: "Challange"
                    },
                    {
                      icon: "fal fa-hand-rock",
                      name: "Protest"
                    },
                    {
                      icon: "fal fa-headphones-alt",
                      name: "Roleplay"
                    },
                    {
                      icon: "fal fa-grin-alt",
                      name: "Meme"
                    },
                    {
                      icon: "fal fa-shopping-cart",
                      name: "Shop"
                    },
                    {
                      icon: "fal fa-desktop",
                      name: "Technology"
                    },
                    {
                      icon: "fal fa-laugh",
                      name: "Fun"
                    },
                    {
                      icon: "fal fa-share-alt",
                      name: "Social"
                    },
                    {
                      icon: "fal fa-laptop",
                      name: "E-Spor"
                    },
                    {
                      icon: 'fal fa-palette',
                      name: 'Design'
                    },
                    {
                      icon: 'fal fa-users',
                      name: 'Community'
                    }
                    ]                
                }
            },
        
            server: {
                id: "876172052070010910",
                invite: "https://discord.gg/3tM553tenS",
                roles: {
                    administrator: "",
                    moderator: "",
                    profile: {
                        sitecreator : "",
                        booster: "",
                        sponsor: "",
                        supporter: "",
                        partnerRole: ""
                    },
                    codeshare: {
                        javascript: "",
                        html: "",
                        substructure: "",
                        bdfd: "", // Bot Designer For Discord
                        fiveInvite: "",
                        tenInvite: "",
                        fifteenInvite: "",
                        twentyInvite: ""
                    },
                    botlist: {
                        developer: "",
                        certified_developer: "",
                        bot: "",
                        certified_bot: "",
                    }
                },
                channels: {
                    codelog: "",
                    login: "",
                    webstatus: "",
                    uptimelog: "",
                    botlog: "",
                    votes: ""
                }
            }
        
        
        }


config.js

        module.exports = {
            bot: {
                token: "ODgzNjk3ODM4MTY3NTgwNjky.YTNtvg.CjVzqYsR_jStt32ZlSkRxhUmzf8",
                prefix: ".",
                owners: ["844244033525514280"],
                mongourl: "mongodb+srv://RegBotListv14:r4YrQOIziCwZso65@regbotlistv14.iisrw.mongodb.net/myFirstDatabase?retryWrites=true&w=majority",
                servers: {
                    token: "ODgzNjk3ODM4MTY3NTgwNjky.YTNtvg.CjVzqYsR_jStt32ZlSkRxhUmzf8",
                    prefix: "."
                }
            },
        
            website: {
                callback: "https://www.regbotlist.ml/callback",
                secret: "2TFmrgDE8RRnj3I5ijLsywwyVTjGeLca",
                clientID: "883697838167580692", // Bot client id.
                tags: ["Moderation", "Fun", "Minecraft","Economy","Guard","NSFW","Anime","Invite","Music","Logging", "Web Dashboard", "Reddit", "Youtube", "Twitch", "Crypto", "Leveling", "Game", "Roleplay", "Utility", "Turkish"],
                languages: [
                    { flag: 'gb', code: 'en', name: 'English' },
                    { flag: 'tr', code: 'tr', name: 'Türkçe' },
                    { flag: 'de', code: 'de', name: 'Deutsch' },
                    { flag: 'az', code: 'az', name: 'Azerbaycan' }
                ],
                servers: {
                    tags: [
                    {
                        icon: "fal fa-code",
                        name: "Development"
                    },
                    {
                        icon: "fal fa-play",
                        name: "Stream"
                    },
                    {
                        icon: "fal fa-camera",
                        name: "Media"
                    },
                    {
                        icon: 'fal fa-building',
                        name: 'Company'
                    },
                    {
                        icon: 'fal fa-gamepad',
                        name: 'Game'
                    },
                    {
                        icon: 'fal fa-icons',
                        name: 'Emoji'
                    },
                    {
                        icon: 'fal fa-robot',
                        name: 'Bot List'
                    },
                    {
                        icon: 'fal fa-server',
                        name: 'Server List'
                    },
                    {
                        icon: 'fal fa-moon-stars',
                        name: 'Turkish'
                    },
                    {
                        icon: 'fab fa-discord',
                        name: 'Support'
                    },
                    {
                        icon: 'fal fa-volume',
                        name: 'Sound'
                    },
                    {
                        icon: 'fal fa-comments',
                        name: 'Chatting'
                    },
                    {
                        icon: 'fal fa-lips',
                        name: 'NSFW'
                    },
                    {
                      icon: "fal fa-comment-slash",
                      name: "Challange"
                    },
                    {
                      icon: "fal fa-hand-rock",
                      name: "Protest"
                    },
                    {
                      icon: "fal fa-headphones-alt",
                      name: "Roleplay"
                    },
                    {
                      icon: "fal fa-grin-alt",
                      name: "Meme"
                    },
                    {
                      icon: "fal fa-shopping-cart",
                      name: "Shop"
                    },
                    {
                      icon: "fal fa-desktop",
                      name: "Technology"
                    },
                    {
                      icon: "fal fa-laugh",
                      name: "Fun"
                    },
                    {
                      icon: "fal fa-share-alt",
                      name: "Social"
                    },
                    {
                      icon: "fal fa-laptop",
                      name: "E-Spor"
                    },
                    {
                      icon: 'fal fa-palette',
                      name: 'Design'
                    },
                    {
                      icon: 'fal fa-users',
                      name: 'Community'
                    }
                    ]                
                }
            },
        
            server: {
                id: "876172052070010910",
                invite: "https://discord.gg/3tM553tenS",
                roles: {
                    administrator: "876172052082610190",
                    moderator: "876172052082610187",
                    profile: {
                        sitecreator : "883693988572450887",
                        booster: "",
                        sponsor: "883694151315640360",
                        supporter: "876172052070010917",
                        partnerRole: "883694239647662122"
                    },
                    codeshare: {
                        javascript: "883694331448406037",
                        html: "883694408250294353",
                        substructure: "883694484100116531",
                        bdfd: "883694551456448557", // Bot Designer For Discord
                        fiveInvite: "883694611875389471",
                        tenInvite: "883694685661569034",
                        fifteenInvite: "883694762505404427",
                        twentyInvite: "883694831375880223"
                    },
                    botlist: {
                        developer: "876172052070010914",
                        certified_developer: "876172052070010915",
                        bot: "876172052070010911",
                        certified_bot: "876172052070010912",
                    }
                },
                channels: {
                    codelog: "876172052590112785",
                    login: "876172052590112785",
                    webstatus: "876172052590112785",
                    uptimelog: "876172052590112785",
                    botlog: "876172052590112785",
                    votes: "876172052590112785"
                }
            }
        }